ctx.go

```diff
@@ -309,6 +311,17 @@ func (c *Ctx) Context() *fasthttp.RequestCtx {
 	return c.fasthttp
 }
 
+func (c *Ctx) UserContext() context.Context {
```

Comment:
add a method docblock with description 

we receive a lot of support requests and i think this could eliminate some requests from the start

Reply:
Done, 
Kindly review